webapp/src/Entity/Contest.php

```diff
@@ -170,6 +170,52 @@ class Contest extends BaseApiEntity
      */
     private $b = 0;
 
+    /**
+     * @var boolean|null
+     * @ORM\Column(type="boolean", name="process_awards",
+     *     options={"comment"="Are there awards for this contest?","default"=0},
```

Comment:
Nitpick: `Whether to process awards for this contest` (or maybe `generate` instead of `process`)

Reply:
Sure.